modules/learn/pages/views/views-querying.adoc

```diff
@@ -537,7 +537,7 @@ Compare this with the order of precedence used in Unicode collation, which is us
 
 Notice again that items that start with integers will appear before any items with letters.
 However, in this case, the lowercase and then uppercase of the same letter are grouped together.
-This means that that if "apple" will appear before "Apple" and would also appear before "Zebra." In addition, be aware that with accented characters will follow this ordering:
+This means that if "apple" will appear before "Apple" and would also appear before "Zebra." In addition, be aware that with accented characters will follow this ordering:
```

Comment:
I think this sentence may need reworking beyond the extra "that" being removed.
Perhaps remove the "if", and change the "would" to another "will"?